controller/rancher/rancher_cert_dir_test.go

```diff
@@ -87,6 +87,22 @@ func TestCheckCertDirUpdate(t *testing.T) {
 		t.Fatalf("Error building config %v", err)
 	}
 
+	if len(newConfigs[0].Certs) != 3 {
```

Comment:
What does this addition to the test assert in relation to the change in the rest of the PR?